Orange/widgets/regression/owsgdregression.py

```diff
@@ -42,7 +40,7 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     learning_rate = settings.Setting(InvScaling)
 
     want_main_area = False
-    resizing_enabled = False
+    resizing_enabled = True
```

Comment:
Not for the reasons above, no. This PR is still about a valid issue, namely reporting in SGD widget. However, now that I look at it, it's a duplicate of https://github.com/biolab/orange3/pull/1107.